src/main/java/org/apache/sling/api/resource/AbstractResourceVisitor.java

```diff
@@ -23,13 +23,13 @@
 import org.jetbrains.annotations.NotNull;
 
 /**
- * This visitor will traverse the given resource and all its children in a breadth-first approach
+ * This visitor will traverse the given resource and all its children in a depth-first approach
  * and call the {@link AbstractResourceVisitor#visit(Resource)} method for each visited resource.
  * It decouples the actual traversal code from application code. 
  * 
  * Concrete subclasses must implement the {@link AbstractResourceVisitor#visit(Resource)} method.
  *
- * @see <a href="https://en.wikipedia.org/wiki/Breadth-first_search">Breadth-First-Search</a>
+ * @see <a href="https://en.wikipedia.org/wiki/Depth-first_search">Depth-First-Search</a>
  * @since 2.2 (Sling API Bundle 2.2.0)
```

Comment:
I would suggest making this part of a different change, it's important enough to stand out.

Reply:
Will commit this separately